### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | yes
| BC breaks?       | yes
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | comma-separated list of tickets fixed by the PR, if any
| License          | MIT

### Description

[Description of the bug or feature]

Rewrite notification

--

### CODE REVIEW

File: src/renderer/services/notification/index.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  }) {
+    let rs
+    let rj
+    let timer = null
+
+    const fragment = document.createElement('div')
+    fragment.innerHTML = template
+      .replace(/\{\{icon\}\}/, INON_HASH[type])
+      .replace(/\{\{title\}\}/, title)
+      .replace(/\{\{message\}\}/, message)
```
Comment (fxha) [inline src/renderer/services/notification/index.js]: I'd use DOMPurify to sanitize `title` and `message`. That is most unlikely but just to make sure that no one can load code into renderer.

**Comment by fxha** [comment]
@Jocs I get a console error message when I click on the close button (`x`)  from the notification box: `Uncaught (in promise) undefined`

**Comment by Jocs** [comment]
@fxha Yeh, Because we called `notice.notify` and it return a promise, it will reject the promise every time when we click (X) button, but we didn't write `catch block` sometimes. and it will show this error.

**Comment by Jocs** [comment]
@fxha I'll release new version this weekend, So is there any problems with your side?

**Comment by fxha** [comment]
No, I don't have a problem with that.

**Pull request merged**